src/modules/TeamsList/index.tsx

```diff
@@ -1,20 +1,26 @@
 import React, { FC, useState } from 'react';
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
 import { useTeamsQuery } from 'hooks/graphql';
 import { Loader, ErrorModal, Pagination } from 'components';
 import { selectUserData } from 'modules/StudentsTable/selectors';
 import { selectCurrCourse } from 'modules/LoginPage/selectors';
 import { StyledTeams } from './styled';
-import { TEAMS_PER_PAGE } from 'appConstants';
+import { TEAMS_PER_PAGE, TOUR_OPENING } from 'appConstants';
 import { Team } from 'types';
 import { TeamListModals, Teams } from './components';
 import { useCommonMutations } from './components/TeamListModals/useCommonMutations';
 import { AdditionalWrapper, ContentPageWrapper } from 'typography';
+import { setIsTourOpen } from 'modules/LoginPage/loginPageReducer';
+import { setTourOpening } from 'modules/LoginPage/loginPageMiddleware';
 
 export const TeamsList: FC = () => {
-  const [page, setPage] = useState<number>(0);
+  const [page, setPage] = useState(0);
   const currCourse = useSelector(selectCurrCourse);
   const userData = useSelector(selectUserData);
+  const dispatch = useDispatch();
+  const userTeam =
+    userData.teams.find((team: Team) => team.courseId === currCourse.id) ??
```

Comment:
The find() method returns the value of the first element in the provided array that satisfies the provided testing function. If no values satisfy the testing function, undefined is returned.
So, do we need to use ?? undefined there?